.env.example

```diff
@@ -22,6 +22,7 @@ GRAPHQL_HOST='localhost:3000'
 GRAPHQL_PROTOCOL='http'
 HOST='localhost'
 INVITATION_SHORTLINK='key_INVITATION_SHORTLINK'
+LOG_ROCKET_ID='id/parabol'
```

Comment:
Kind reminders:
1. Update this key & value in action-devops/environments
2. Manually set the value with `dokku` on different stages